{
   "CVE_data_meta" : {
      "ASSIGNER" : "lpardo@redhat.com",
      "ID" : "CVE-2018-10882",
      "STATE" : "PUBLIC"
   },
   "affects" : {
      "vendor" : {
         "vendor_data" : [
            {
               "product" : {
                  "product_data" : [
                     {
                        "product_name" : "kernel",
                        "version" : {
                           "version_data" : [
                              {
                                 "version_value" : "n/a"
                              }
                           ]
                        }
                     }
                  ]
               },
               "vendor_name" : "[UNKNOWN]"
            }
         ]
      }
   },
   "data_format" : "MITRE",
   "data_type" : "CVE",
   "data_version" : "4.0",
   "description" : {
      "description_data" : [
         {
            "lang" : "eng",
            "value" : "A flaw was found in the Linux kernel's ext4 filesystem. A local user can cause an out-of-bound write in in fs/jbd2/transaction.c code, a denial of service, and a system crash by unmounting a crafted ext4 filesystem image."
         }
      ]
   },
   "impact" : {
      "cvss" : [
         [
            {
               "vectorString" : "4.8/CVSS:3.0/AV:P/AC:H/PR:N/UI:N/S:U/C:N/I:L/A:H",
               "version" : "3.0"
            }
         ]
      ]
   },
   "problemtype" : {
      "problemtype_data" : [
         {
            "description" : [
               {
                  "lang" : "eng",
                  "value" : "CWE-787"
               }
            ]
         }
      ]
   },
   "references" : {
      "reference_data" : [
         {
            "name" : "[debian-lts-announce] 20180718 [SECURITY] [DLA 1423-1] linux-4.9 new package",
            "refsource" : "MLIST",
            "url" : "https://lists.debian.org/debian-lts-announce/2018/07/msg00020.html"
         },
         {
            "name" : "https://bugzilla.kernel.org/show_bug.cgi?id=200069",
            "refsource" : "CONFIRM",
            "url" : "https://bugzilla.kernel.org/show_bug.cgi?id=200069"
         },
         {
            "name" : "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10882",
            "refsource" : "CONFIRM",
            "url" : "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10882"
         },
         {
            "name" : "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=c37e9e013469521d9adb932d17a1795c139b36db",
            "refsource" : "CONFIRM",
            "url" : "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=c37e9e013469521d9adb932d17a1795c139b36db"
         },
         {
            "name" : "RHSA-2018:2948",
            "refsource" : "REDHAT",
            "url" : "https://access.redhat.com/errata/RHSA-2018:2948"
         },
         {
            "name" : "USN-3753-1",
            "refsource" : "UBUNTU",
            "url" : "https://usn.ubuntu.com/3753-1/"
         },
         {
            "name" : "USN-3753-2",
            "refsource" : "UBUNTU",
            "url" : "https://usn.ubuntu.com/3753-2/"
         }
      ]
   }
}
